boards/DCM/Src/Io/Io_IsrHalCallback.c

```diff
@@ -0,0 +1,15 @@
+#include <stm32f3xx_hal.h>
+#include "main.h"
+#include "Io_LSM6DS33.h"
```

Comment:
This file should be renamed to Io_ExternalInterruptCallback.c

Reply:
ok